src/analysis/normed_space/linear_isometry.lean

```diff
@@ -586,6 +586,20 @@ rfl
   ((prod_assoc R E E₂ E₃).symm : E × E₂ × E₃ → (E × E₂) × E₃) = (equiv.prod_assoc E E₂ E₃).symm :=
 rfl
 
+variables {R E E₂ E₃} {R' : Type*} [ring R'] [module R' E] (p q : submodule R' E)
+
+/-- `linear_equiv.of_eq` as a `linear_isometry_equiv`. -/
+def of_eq (hpq : p = q) :
+  p ≃ₗᵢ[R'] q :=
+{ norm_map' := λ x, rfl,
+  ..linear_equiv.of_eq p q hpq }
+
+variables {p q}
+
+@[simp] lemma coe_of_eq_apply (h : p = q) (x : p) : (of_eq p q h x : E) = x := rfl
```

Comment:
Could you please setup `simps` to work with linear isometries, if that is not already done? Then these lemmas can be auto-generated.

Reply:
I don't think simps is easy to configure to generate the right name here (with `coe` as a prefix)

Reply:
Yes, if we want to autogenerate these we can't do anything about the names. I'm struggling a bit to understand how to setup these properly, but I'll ask people in Providence tomorrow

Reply:
I did the `simps` setup and removed the `coe_of_eq_apply`, here and for `linear_equiv`, since `simps` generate it as `of_eq_apply`, which is more homogeneous with the rest of the library. Does that sound fine ?